### PR TITLE
DONT MERGE! Revert "Revert "Fix formatting to fix admonitions in 1.25 docs""

### PR DESCRIPTION
**Pull request by evilnick**
Reverts juju/docs#2842

### CODE REVIEW

**Comment by evilnick** [comment]
@pmatulis Travis is your friend. It wasn't enabled on the old 1.25 docs before, but it is now

File: src/en/juju-backups.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ configuration files (such as '.jenv' files for running environments, or
 credentials) it is usually best to simply back up the entire Juju directory
 (`~/.juju`).
 
-!!! Note: On Windows systems, the Juju directory is in a different place
- (usually `C:\Users\<username>\AppData\Roaming\Juju`. Also, although `tar` is 
-available for Windows, you may prefer to use a more Windows-centric backup 
-application.
+!!! Note: 
+    On Windows systems, the Juju directory is in a different place
+     (usually `C:\Users\<username>\AppData\Roaming\Juju`. Also, although `tar` is
```
Comment (evilnick) [inline src/en/juju-backups.md]: probably this

Comment (pmatulis) [reply src/en/juju-backups.md]: I updated that paragraphs but the failure remains. Still looking.

**Comment by pmatulis** [comment]
I removed this entire admonishment and the Travis CI test passed.

```
    !!! Note: 
    On Windows systems, the Juju directory is in a different place
    (usually `C:\Users\<username>\AppData\Roaming\Juju`. Also, although
    `tar` is available for Windows, you may prefer to use a more
    Windows-centric backup application.

**Comment by evilnick** [comment]
Although the backticks imply a code style, in admonitions I think they get
processed last. Pretty sure it's either the < > or \

**Comment by pmatulis** [comment]
I changed `<username>` to `USERNAME` and the test passed.

**Pull request merged**